auth.py

```diff
@@ -59,36 +60,37 @@ def login():
     log = current_app.logger
 
     if current_user.is_authenticated:
-        return redirect(url_for('profile'))
+        return redirect(url_for('root'))
 
-    form = LoginForm(request.form)
+    form = LoginForm()
```

Comment:
i don't think this will work as the form is not created with `request.form`

Reply:
it works! I tested, but either way I think it should work! I'll add it in.